downstream/modules/platform/ref-controller-notification-webhook.adoc

```diff
@@ -0,0 +1,37 @@
+[id="controller-notification-webhook"]
+
+= Webhook
+
+The webhook notification type provides a simple interface for sending *POSTs* to a predefined web service.
```

Comment:
Is POST an acronym?

Reply:
No it is a HTTP method:
https://www.w3schools.com/tags/ref_httpmethods.asp

Maybe it should be enclosed by backticks rather than bolded.